test/unit/storage/accessors/FilterMetadataDataAccessor.test.ts

```diff
@@ -0,0 +1,107 @@
+import { APPLICATION_JSON, CONTENT_LENGTH_TERM, CONTENT_TYPE_TERM, toNamedTerm } from '../../../../src';
+import { RepresentationMetadata } from '../../../../src/http/representation/RepresentationMetadata';
+import type { DataAccessor } from '../../../../src/storage/accessors/DataAccessor';
+import { FilterPattern,
+  FilterMetadataDataAccessor } from '../../../../src/storage/accessors/FilterMetadataDataAccessor';
+import { guardedStreamFrom } from '../../../../src/util/StreamUtil';
+
+describe('FilterMetadataDataAccessor', (): void => {
+  let childAccessor: jest.Mocked<DataAccessor>;
+
+  const mockIdentifier = { path: 'http://localhost/test.txt' };
+  const mockData = guardedStreamFrom('test string');
+
+  beforeEach(async(): Promise<void> => {
+    jest.clearAllMocks();
+    childAccessor = {
+      writeDocument: jest.fn(),
+      writeContainer: jest.fn(),
```

Comment:
Since we have the mocked functions already, would also add tests everywhere to test that these have been called and with the correct parameters.

E.g.
```ts
expect(writeDocument).toHaveBeenCalledTimes(1);
expect(writeDocument).toHaveBeenLastCalledWith(mockIdentifier, mockData, metadata);
```
etc.